string aleaType
string data
---
bool success
